block_linux.go

```diff
@@ -310,6 +310,19 @@ func (ctx *context) diskPartitions(disk string) []*Partition {
 	return out
 }
 
+func (ctx *context) isRemovable(disk string) bool {
```

Comment:
suggest naming the method `diskIsRemovable()` just to match the other methods like `diskPartitions()` or `diskIsRotational()`